evalml/model_understanding/graphs.py

```diff
@@ -77,9 +77,53 @@ def normalize_confusion_matrix(conf_mat, normalize_method='true'):
             raise ValueError('Invalid value provided for "normalize_method": %s'.format(normalize_method))
         if w and "invalid value encountered in" in str(w[0].message):
             raise ValueError("Sum of given axis is 0 and normalization is not possible. Please select another option.")
+    conf_mat = pd.DataFrame(conf_mat, index=col_names, columns=col_names)
     return conf_mat
 
 
+def graph_confusion_matrix(y_true, y_pred, normalize_method='true', title_addition=None):
```

Comment:
Just moved this up, to be spatially connected with the other confusion matrix-related methods 😆